.github/workflows/ci.yml

```diff
@@ -258,7 +269,7 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       contents: write
-    needs: [linux, macos, windows, dist, docker]
+    needs: [linux, macos, windows, dist]
```

Comment:
We should abort releasing if docker build/publish fails. Even if this change releases the artifact quickly, this isn't good.

Reply:
To clarify, the `release` job creates a draft release instead of releasing it. However, I understand you want to wait for it before creating the draft release. I put it back, provided that you will improve the docker job speed by adopting @userdocs's suggestions.